package.json

```diff
@@ -0,0 +1,29 @@
+  {
+    "name": "geotiff-tilesource",
+    "version": "0.2.0",
+    "description": "",
+    "files": [
+      "dist"
+    ],
+    "main": "./dist/GeoTIFFTileSource.umd.js",
+    "module": "./dist/GeoTIFFTileSource.es.js",
+    "exports": {
+      ".": {
+        "import": "./dist/GeoTIFFTileSource.es.js",
+        "require": "./dist/GeoTIFFTileSource.umd.js"
+      }
+    },
+    "scripts": {
+      "dev": "vite",
+      "build": "vite build",
+      "preview": "vite preview"
+    },
+    "type": "module",
+    "devDependencies": {
+      "prettier": "3.2.5",
+      "vite": "^5.0.8"
+    },
+    "dependencies": {
```

Comment:
Should OpenSeadragon be listed as a dependency too?

Reply:
@abiswas97 thoughts on this? I feel like listing openseadragon here would make sense.